packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -15,21 +15,24 @@ import { Construct } from '@aws-cdk/core';
  */
 export interface KafkaEventSourceProps extends StreamEventSourceProps {
   /**
-   * the Kafka topic to subscribe to
+   * The Kafka topic to subscribe to
    */
   readonly topic: string,
   /**
-   * the secret with the Kafka credentials, see https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html for details
+   * The secret with the Kafka credentials, see https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html for details
+   * This field is required if your Kafka brokers are accessed over the Internet
```

Comment:
Is there a way to detect this state and error? 
For instance, if the lambda is not attached to a vpc. Wouldn't this imply that the access will be over the internet?

Reply:
Added an error on line 161.